Gemfile.lock

```diff
@@ -373,7 +373,8 @@ GEM
     net-ssh-gateway (2.0.0)
       net-ssh (>= 4.0.0)
     netrc (0.11.0)
-    newrelic_rpm (8.16.0)
+    newrelic_rpm (9.6.0)
```

Comment:
I think we might want to do 9.7.1 -- according to the [releases page](https://github.com/newrelic/newrelic-ruby-agent/releases), it looks like that's the current release now. (9.6.0 was the current version when the Jira issue was made in November.)

Reply:
Oki, let me change it today